### ISSUE TITLE
ServiceBus Trigger requires SharedAccessKey with Manage claim

### ISSUE THREAD

**Issue by ChRonX13** [Azure/Azure-Functions#351]
Unsure if this is intended or an issue with the tooling. Was hoping someone could shed some light on this.

We have previously setup our Azure Service Bus to have multiple shared access policies, namely one where we have a Send claim and one where we have a Listen claim. For our current use case, we were hoping to create a Function with a ServiceBus Topic Trigger to listen for events that we wished to process. As such, we had assumed that using the ConnectionString with only a 'listen' claim would be sufficient.

Upon deploying this project, it appears that we are getting errors on the Azure Functions portal with the following message:
![image](https://user-images.githubusercontent.com/4549796/27365997-e9a2010e-5675-11e7-81f9-9f1a00edc10e.png)

Upon testing with the root access policy, things seem to work fine.

Is the above scenario intended behaviour? or a possible bug in the tooling? We currently have no requirement to send or manage service bus functionality with this Function app.

Thanks

**Comment by lindydonna** [comment]
CC @tohling

**Comment by tohling** [comment]
@ChRonX13, this is a known issue that has been fixed but has not been rolled out to production.  Kindly see https://github.com/Azure/azure-webjobs-sdk-script/issues/1048#issuecomment-314893017

**Comment by bravegeek** [comment]
Thanks for the work on this!
Has this been rolled out to production?
If so, how do I take advantage of it?

**Comment by tohling** [comment]
@bravegeek , yes this is now pushed to production.  You should now be able to use a connection string that has either "listen" or "manage" access rights.
